common/script/statHelpers.js

```diff
@@ -0,0 +1,55 @@
+/*
+  ------------------------------------------------------
+  Level cap
+  ------------------------------------------------------
+ */
+
+const maxLevel = 100;
+
+function capByLevel (lvl) {
+  if (lvl > maxLevel) {
+    return maxLevel;
+  } else {
+    return lvl;
+  }
+}
+
+/*
+  ------------------------------------------------------
+  Health cap
+  ------------------------------------------------------
+ */
+
+const maxHealth = 50;
+
+/*
+  ------------------------------------------------------
+  Scoring
+  ------------------------------------------------------
+ */
+
+function tnl (lvl) {
+  return Math.round((Math.pow(lvl, 2) * 0.25 + 10 * lvl + 139.75) / 10) * 10;
```

Comment:
There are so many magic numbers here. Let's split them out and make them intelligible.